src/Pages/OrderReviewPage.js

```diff
@@ -39,107 +38,109 @@ class OrderItem {
 }
 
 function OrderReviewPage() {
+  console.log("Render Page");
+
   /*
         De hoofddata moet aangepast worden door op de + en - te klikken 
         in de OrderItemCard componenten.
         Dit moet zichtbaar zijn in de UI wanneer er op die knoppen
         gedrukt wordt. Dit werkt momenteel niet. 
     */
 
+  const orderRef = useRef([]);
   const [order, setOrder] = useState({ orderItems: [] });
-  const selectedItemIds = useContext(CheckoutContext).selectedItems;
-  const AddMenuItemToOrder = useContext(CheckoutContext).AddMenuItemToOrder;
-  const SubtractItemFromOrder =
-    useContext(CheckoutContext).SubtractItemFromOrder;
 
   useEffect(() => {
-    if (order.orderItems.length < 1 /* Should run only the first time */) {
```

Comment:
this overrides a lot of  changes about sending orders from the menu page to the review page